src/box/index.cc

```diff
@@ -146,31 +146,22 @@ key_validate(const struct index_def *index_def, enum iterator_type type,
 }
 
 int
-exact_key_validate(struct key_def *key_def, const char *key,
+exact_key_validate(struct index_def *index_def, const char *key,
```

Comment:
Bad commit subject: sounds like this function just drops something unused while in fact it does quite a bit of refactoring. I'd say something like "box: refactor index point lookup validation"

Reply:
Done.